### PR TITLE
Mg greening

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

File: cypress/support/fragments/data_import/job_profiles/jobProfiles.js [hunk 1]
```diff
@@ -57,6 +57,16 @@ const createJobProfile = (jobProfile) => {
   newJobProfile.fillJobProfile(jobProfile);
 };
 
+const searchJobProfile = (profileName) => {
+  cy.do(TextField({ id:'input-search-job-profiles-field' }).fillIn(profileName));
+  cy.do(Pane({ id:'pane-results' }).find(Button('Search')).click());
+};
+
+const waitCreatingJobProfile = () => {
+  cy.expect(Pane({ id:'pane-results' }).find(Button('Actions')).exists());
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/job_profiles/jobProfiles.js:66]: move to const

File: cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js [hunk 2]
```diff
@@ -180,7 +180,7 @@ export default {
       Select({ name:'profile.incomingRecordType' }).choose(incomingRecordType.edifact),
       Select({ name:'profile.existingRecordType' }).choose(folioRecordTypeValue.invoice),
       TextArea({ name:'profile.description' }).fillIn(''),
-      TextField('Batch group*').fillIn('"FOLIO"'),
+      TextField('Batch group*').fillIn('"Amherst (AC)"'),
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js:183]: Does need double "? maybe '' or ""

File: cypress/support/fragments/inventory/inventoryItem/inventoryItems.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    cy.do(Button({ id: `accordion-toggle-button-${holdingId}` }).click());
+  },
+
+  verifyItemWithStatusExists(holdingId, status) {
+    cy.expect(MultiColumnList({ id: `list-items-${holdingId}` }).find(MultiColumnListCell(status)).exists());
+  },
+
+  searchByParameter: (parameter, value) => {
+    cy.do(SearchField({ id: 'input-inventory-search' }).selectIndex(parameter));
+    cy.do(SearchField({ id: 'input-inventory-search' }).fillIn(value));
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryItem/inventoryItems.js:40]: move to const

File: cypress/support/fragments/inventory/inventoryItem/inventoryItems.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+export default {
+  toggleStatus(statusName) {
+    cy.do(Accordion({ id:'itemStatus' }).find(TextField({ type:'search' })).fillIn(statusName));
+    // interactor doesn't click
+    cy.get('#itemStatus input').click();
+    cy.do(Checkbox(statusName).click());
+  },
+
+  toggleItemStatusAccordion() {
+    cy.do(Accordion({ id:'itemStatus' }).clickHeader());
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryItem/inventoryItems.js:23]: move to const

File: cypress/support/fragments/inventory/inventoryItem/inventoryItems.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/support/fragments/inventory/inventoryItem/inventoryItems.js:40]: Done

File: cypress/support/fragments/data_import/job_profiles/jobProfiles.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/job_profiles/jobProfiles.js:66]: Done

File: cypress/support/fragments/inventory/inventoryItem/inventoryItems.js (hunk 4 above)
Comment (TetianaParanich) [reply cypress/support/fragments/inventory/inventoryItem/inventoryItems.js:23]: Done

File: cypress/integration/data-import/match-on-pol-and-update-items.spec.js [hunk 5]
```diff
@@ -259,13 +270,12 @@ describe('ui-data-import: Match on POL and update related Instance, Holdings, It
         materialTypeId
       ));
     Orders.checkIsOrderCreated(firstItem.orderNumber);
-
     // open the first PO
     openOrder(firstItem.orderNumber);
     OrderView.checkIsOrderOpened('Open');
     OrderView.checkIsItemsInInventoryCreated(firstItem.title, 'Main Library');
```
Comment (penfriend) [inline cypress/integration/data-import/match-on-pol-and-update-items.spec.js:276]: Are you sure that 'Main Library' exists?

File: cypress/integration/data-import/match-on-vendor-reference-number.spec.js [hunk 6]
```diff
@@ -85,11 +85,11 @@ describe('ui-data-import: Match on VRN and update related Instance, Holdings, It
     });
   });
 
-  after(() => {
+  /*after(() => {
     Users.deleteViaApi(userId);
     MatchOnVRN.deletePOLineViaAPI(MatchOnVRN.poLineData.title);
     MatchOnVRN.deleteItemViaAPI();
-  });
+  });*/
```
Comment (penfriend) [inline cypress/integration/data-import/match-on-vendor-reference-number.spec.js:92]: Do you need this comment?

File: cypress/support/fragments/data_import/job_profiles/jobProfiles.js [hunk 7]
```diff
@@ -57,6 +58,16 @@ const createJobProfile = (jobProfile) => {
   newJobProfile.fillJobProfile(jobProfile);
 };
 
+const searchJobProfile = (profileName) => {
+  cy.do(TextField({ id:'input-search-job-profiles-field' }).fillIn(profileName));
+  cy.do(resultsPane.find(Button('Search')).click());
```
Comment (penfriend) [inline cypress/support/fragments/data_import/job_profiles/jobProfiles.js:63]: you can write cy.do( [ ]) one time

File: cypress/integration/data-import/match-on-vendor-reference-number.spec.js (hunk 6 above)
Comment (TetianaParanich) [reply cypress/integration/data-import/match-on-vendor-reference-number.spec.js:92]: I deleted it

File: cypress/support/fragments/inventory/inventoryItem/inventoryItems.js [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+  selectInstance(title) {
+    cy.do(MultiColumnListCell(title).click());
+  },
+
+  toggleAccordionItemsButton(holdingId) {
+    cy.do(Button({ id: `accordion-toggle-button-${holdingId}` }).click());
+  },
+
+  verifyItemWithStatusExists(holdingId, status) {
+    cy.expect(MultiColumnList({ id: `list-items-${holdingId}` }).find(MultiColumnListCell(status)).exists());
```
Comment (penfriend) [inline cypress/support/fragments/inventory/inventoryItem/inventoryItems.js:37]: Are you sure that this method actually verifies the status?

File: cypress/support/fragments/inventory/inventoryItem/inventoryItems.js [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+  },
+
+  searchByParameter: (parameter, value) => {
+    cy.do(searchField.selectIndex(parameter));
+    cy.do(searchField.fillIn(value));
+    cy.do(searchButton.focus());
+    cy.do(searchButton.click());
```
Comment (penfriend) [inline cypress/support/fragments/inventory/inventoryItem/inventoryItems.js:44]: cy.do ( [  ])

File: cypress/support/fragments/orders/orderLines.js [hunk 10]
```diff
@@ -256,6 +257,10 @@ export default {
 
   openInstance:() => {
     cy.do(PaneContent({ id:'order-lines-details-content' }).find(Link({ href: including('/inventory/view/') })).click());
+  },
+
+  openReceiving:() => {
+    cy.do(PaneHeader({ id: 'paneHeaderorder-lines-details' }).find(actionsButton).click());
+    cy.do(Button('Receive').click());
```
Comment (penfriend) [inline cypress/support/fragments/orders/orderLines.js:264]: cy.do ( [])

File: cypress/support/fragments/data_import/job_profiles/jobProfiles.js (hunk 7 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/job_profiles/jobProfiles.js:63]: Done

File: cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js:183]: I need string like "Amherst (AC)"

File: cypress/integration/data-import/match-on-pol-and-update-items.spec.js (hunk 5 above)
Comment (TetianaParanich) [reply cypress/integration/data-import/match-on-pol-and-update-items.spec.js:276]: yes, I got it 'cy.getLocations({ limit:1, query:'name="Main Library"' })'

File: cypress/support/fragments/inventory/inventoryItem/inventoryItems.js (hunk 8 above)
Comment (TetianaParanich) [reply cypress/support/fragments/inventory/inventoryItem/inventoryItems.js:37]: yep)

File: cypress/support/fragments/inventory/inventoryItem/inventoryItems.js (hunk 9 above)
Comment (TetianaParanich) [reply cypress/support/fragments/inventory/inventoryItem/inventoryItems.js:44]: Done

File: cypress/integration/data-import/match-on-pol-and-update-items.spec.js (hunk 5 above)
Comment (penfriend) [reply cypress/integration/data-import/match-on-pol-and-update-items.spec.js:276]: I mean that in environment this location can be not exists. If you need this location you need to create it

File: cypress/support/fragments/orders/orderLines.js (hunk 10 above)
Comment (TetianaParanich) [reply cypress/support/fragments/orders/orderLines.js]: Done

File: cypress/integration/data-import/match-on-vendor-reference-number.spec.js [hunk 11]
```diff
@@ -16,7 +16,7 @@ import DevTeams from '../../support/dictionary/devTeams';
 describe('ui-data-import: Match on VRN and update related Instance, Holdings, Item', () => {
```
Comment (IhorBohdan) [inline cypress/integration/data-import/match-on-vendor-reference-number.spec.js:16]: Fix linter issues

**Review by IhorBohdan — APPROVED** [review]

File: cypress/support/fragments/orders/orderLines.js [hunk 12]
```diff
@@ -15,7 +15,8 @@ import {
   SelectionList,
   Link,
   including,
-  PaneContent
+  PaneContent,
```
[the remark this answers is not in the document]
Comment (TetianaParanich) [reply cypress/support/fragments/orders/orderLines.js:18]: Done

File: cypress/integration/data-import/match-on-pol-and-update-items.spec.js (hunk 5 above)
Comment (TetianaParanich) [reply cypress/integration/data-import/match-on-pol-and-update-items.spec.js]: I created location

File: cypress/integration/data-import/match-on-vendor-reference-number.spec.js (hunk 11 above)
Comment (TetianaParanich) [reply cypress/integration/data-import/match-on-vendor-reference-number.spec.js:15]: Done

**Review by penfriend — APPROVED** [review]

**Pull request merged**